### ISSUE TITLE
Training the SUNRGBD dataset, the effect is very poor

### ISSUE THREAD

**Issue by limingyue123** [YunYang1994/tensorflow-yolov3#323]
After converting the SUNRGBD dataset format to VOC format, I trained 30 epochs with this code, and the loss dropped to 8.8 or so. Then I tested it and got 10% mAP. I looked at the data and there is no problem. You Can you provide a solution? I am looking for your reply at 599199268@qq.com.
@YunYang1994
